.github/docker/setup-s3c.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+# setup S3C environment just like
+# https://github.com/scality/Integration/blob/development/9.5/tests/setup-environment/index.js
+
+set -e -o pipefail
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+VAULTCLIENT=$SCRIPT_DIR/../../node_modules/vaultclient/bin/vaultclient
+CONFIG=$SCRIPT_DIR/admin.json
+
+echo "Deleting and setting up S3C accounts like Integration (follow conf/authdata.json)"
+
+$VAULTCLIENT --config $CONFIG delete-account --name Bart || true
+$VAULTCLIENT --config $CONFIG create-account \
+    --name Bart \
+    --email sampleaccount1@sampling.com \
+    --accountid 123456789012 \
+    --canonicalid 79a59df900b949e55d96a1e698fbacedfd6e09d98eacf8f8d5218e7cd47ef2be
+$VAULTCLIENT --config $CONFIG generate-account-access-key \
+    --name Bart \
+    --accesskey ACC1AK00000000000000 \
```

Comment:
Since we are specifying those credentials in `s3c_credentials.json` should we read from it?

Reply:
Not necessarily, I've hardcoded here so we can easily rerun manually and quickly troubleshoot without having to look for creds. Same for Integration, it will be hardcoded